### PR TITLE
Commerce App Service

### PR DESCRIPTION
**Pull request by khburdette**
We want to add a new app service that can provide a better search experience by letting other apps feed data into a search

### CODE REVIEW

**Comment by khburdette** [comment]
@theresalech This PR is ready for review

File: proposals/NNNN-Commerce-App-Service.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+This new RPC is necessary for a consumer app to be able to search for POIs.
+
+```xml
+<element name="POISearchID" value="??" hexvalue="??" since="??" />
+
+<function name="POISearch" functionID="SendPOIsID" messagetype="request">
+	<description> This request is sent to the module to send POIs. </description>
+
+    <param name="keywords" type="String" array="true" mandatory="true">
+        <description> This is an arrary of search words that the user is searching for. </description>
```
Comment (theresalech) [inline proposals/NNNN-Commerce-App-Service.md]:
```suggestion
        <description> This is an array of search words that the user is searching for. </description>
```

File: proposals/NNNN-Commerce-App-Service.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    </param>
+
+    <param name="searchCenter" type="GPSData" mandatory="true">
+        <description> This is center of where the user is searching. </description>
```
Comment (theresalech) [inline proposals/NNNN-Commerce-App-Service.md]:
```suggestion
        <description> This is the center of where the user is searching. </description>
```

File: proposals/NNNN-Commerce-App-Service.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+    </param>
+
+    <param name="searchRadius" type="Float" mandatory="true">
+        <description> This is radius of where to search. </description>
```
Comment (theresalech) [inline proposals/NNNN-Commerce-App-Service.md]:
```suggestion
        <description> This is the radius of where to search. </description>
```

**Comment by khburdette** [comment]
@theresalech I've accepted all the comments, thanks!

**Pull request merged**